website/src/content/api-reference/application-config.mdx

```diff
@@ -355,6 +355,78 @@ Using `manage_` OAuth Scopes always imply the corresponding `view_` OAuth Scope.
 
 </Info>
 
+## `additionalOAuthScopes`
+
+The configuration for additional OAuth Scopes and user permissions extending [oAuthScopes configuration](#oauthscopes).
```

Comment:
I'm missing some more context about what this field enables Custom Application developers to achieve.
Perhaps that's something to include in the [OAuth Scopes and user permissions](https://docs.commercetools.com/custom-applications/concepts/oauth-scopes-and-user-permissions) page.

Also, maybe it's worthy to add a new release note explaining this new feature.

Reply:
Adding some explanation in the [OAuth Scopes and user permissions](https://docs.commercetools.com/custom-applications/concepts/oauth-scopes-and-user-permissions) page is a very good point 👍